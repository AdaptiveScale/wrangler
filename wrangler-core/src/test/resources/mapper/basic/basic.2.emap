// This test is adding a default value to output field as well creating a field by concatinating other fields.
out.rec2a = in.rec1.rec2.a + ':' + in.b;
out.x = in.f;
out.f = 3.4;