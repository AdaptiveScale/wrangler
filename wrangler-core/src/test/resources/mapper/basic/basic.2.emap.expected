{"a":null,"b":null,"c":null,"d":null,"e":null,"f":3.4,"x":4.56,"rec2a":"rec2-test-a:sub-test","rec3":[]}